### ISSUE TITLE
Errors not reported by tslint reported by vscode-tslint in 2.0.0

### ISSUE THREAD

**Issue by ashleahhill** [Microsoft/vscode-tslint#74]
I am using the this plugin to aid in developing an Angular2 app. SUPER HELPFUL.

Unfortunately, after upgrading to Typescript 2.0.0 and adding some new configuration options to my tsconfig.json, vscode-tslint is no longer reporting correctly. 

Now, vscode-tslint is not recognizing that I have the `experimentalDecorators` option turned on. It is also not recognizing global typings (like `require`) . tslint itself reports no issues.

## Example Errors
```
[ts]
Experimental support for decorators is a feature that is subject to change in a future release. Set the 'experimentalDecorators' option to remove this warning.

[ts]
Cannot find name 'require'.
```

## Environment

* global tslint: not installed
* tslint version: 3.15.1
* typescript version: 2.0.0
* vscode-tslint version: 0.5.34 
* Visual Studio Code version: 1.4.0

### Previous

```json
{
  "compilerOptions": {
    "target": "ES5",
    "module": "commonjs",
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "sourceMap": true,
    "noEmitHelpers": true
  },
  "exclude": [
    "node_modules",
    "typings/main",
    "typings/main.d.ts"
  ],
  "compileOnSave": false,
  "buildOnSave": false
}
```

### Current

```json
{
  "compilerOptions": {
    "target": "es5",
    "module": "commonjs",
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "sourceMap": true,
    "noEmitOnError": true,
    "declaration": true,
    "lib": [
      "es6",
      "es2015",
      "dom"
    ],
    "moduleResolution": "node",
    "outDir": "lib",
    "rootDir": "./src/components",
    "inlineSources": true,
    "stripInternal": true,
    "typeRoots": [
      "./node_modules/@types"
    ],
    "types": [
      "node"
    ]
  },
  "include": [
    "./src/components/**/*.ts"
  ],
  "exclude": [
    "./**/*.spec.ts"
  ]
}
```

**Comment by egamma** [comment]
When you install vscode-tslint then you get diagnostics from both the TypeScript language service and from tslint: 
- errors/warnings with the prefix `[ts]` come from the TypeScript language service 
- errors/warnigs with the prefix `[tslint]` come from tslint.

The errors you show have nothing to do with tslint. 

>[ts] Experimental support for decorators is a feature that is subject to change in a future release. 

A good test to verify the tslint setup is to call tslint from the command line

So it looks like you have configuration issues with the TypeScript 2.0 project in general.

Can you share the project so that we can help to track this down?

**Comment by ashleahhill** [comment]
Thank you for clarifying what the errors mean.

Unfortunately I can't share the project. I can give you more details about the setup of the project if you know what would be helpful to know?

TypeScript compiles fine in the project using the "Current" tsconfig.json above. Tslint also passes everything. I think the issue might be with VSCode itself not using the correct version of TypeScript to parse my project, (1.8.0 instead of 2.0.0).

**Comment by egamma** [comment]
In this case I suspect that you did not configure vscode to use TypeScript 2.0 using the `tsdk` setting?

Please see https://code.visualstudio.com/docs/languages/typescript#_using-newer-typescript-versions

**Comment by ashleahhill** [comment]
That fixed it! Thank you!

**Comment by egamma** [comment]
your welcome.
